src/AssetManager/View/Helper/Asset.php

```diff
@@ -0,0 +1,114 @@
+<?php
+namespace AssetManager\View\Helper;
+
+use AssetManager\Exception\InvalidArgumentException;
+use Zend\ServiceManager\ServiceLocatorInterface;
+use Zend\View\Helper\AbstractHelper;
+use Zend\Cache\Storage\Adapter\AbstractAdapter as AbstractCacheAdapter;
+
+class Asset extends AbstractHelper
+{
+    /**
+     * @var array
+     */
+    private $config;
+
+    /**
+     * @var ServiceLocatorInterface
+     */
+    private $serviceLocator;
+
+    /**
+     * @param ServiceLocatorInterface $serviceLocator
+     */
+    public function __construct(ServiceLocatorInterface $serviceLocator)
```

Comment:
Don't inject the service locator. Inject the config from the asset manager, as well as the asset manager itself.